### PR TITLE
Prevent application from closing when user cancels file selection

### PR DESCRIPTION
**Pull request by cipherboy**
Closes: #169 

Also clarifies that closing the file will lose the tailoring.

### CODE REVIEW

File: src/MainWindow.cpp [hunk 1]
```diff
@@ -402,6 +402,9 @@ void MainWindow::openFileDialog()
     if (!QFileInfo(defaultDirectory).isDir())
         defaultDirectory = "";
 
+    // Save the current file path to restore if we're unable to open a valid file
+    QString current_file = getOpenedFilePath();
```
Comment (mpreisler) [inline src/MainWindow.cpp]: code style, camelCase please

Comment (mpreisler) [reply src/MainWindow.cpp]: also, make this const

**Comment by cipherboy** [comment]
@mpreisler Updated! :)

**Comment by cipherboy** [comment]
@mpreisler Updated! :)

**Comment by mpreisler** [comment]
Took 7.5 sec on fedora <-- ccache rules

**Pull request merged**